db_test.go

```diff
@@ -122,7 +112,7 @@ func TestUsers(t *testing.T) {
 		t.Errorf("want user to have 1 credentials, got %d", len(user.Credentials))
 	}
 
-	if err := db.UpdateUserCredential(user.ID, user.Credentials["test"]); err != nil {
+	if err := db.UpdateUserCredential(user.ID, user.Credentials["test"].Credential); err != nil {
```

Comment:
Since we have custom type now, I think `UpdateUserCredential` should take a `WebauthnCredential` (which has the user provided ID now). Same for CreateUserCredential, it doesn't need the `name string` param anymore.

Reply:
Actually do we even need `UpdateUserCredential` anymore? Figured you'd always add a new one, verify, then delete the old one rather than edit in place.